### ISSUE TITLE
some error in UniMER dataset

### ISSUE THREAD

**Issue by ManiiXu** [opendatalab/UniMERNet#2]
I found that in the latex results of formula recognition, there is a small probability of non-existent characters such as \delta. After careful comparison with the training dataset, I found that for some special font characters, the corresponding PNG images are not rendered successfully:
For example, images/0921794. png: "{\ mathrm {\ Delta} l} _ {i}=\ mathrm {\ Delta} l ",does not have \Delta in PNG,
Other examples such as \ boldmath~\ omega~(0979049. png), \ boldmath~\ lambda~(1018942. png), etc., may have an impact on the training effectiveness of the model. It is hoped that they can be fixed.

**Comment by ManiiXu** [comment]
When I discovered that there were non-existent characters such as \delta in the test formula, I used a script to extract all the images with that character and quickly looked at a part of them. 
I found that 1) there were missing characters in the images for the corresponding Greek letters; 2) Images like \boldmath, which cannot be rendered on many LaTeX renderers, may also missing the content after boldmath; 3) Apart from \mathrm, \mathbf seems to have corresponding issues as well

**Comment by wangbinDL** [comment]
@ManiiXu We have identified and resolved the issue causing the rendering problems you highlighted. It appears that when Greek letters are enclosed within \mathbf or \mathrm environments, xelatex struggles to render them correctly. To address this, we have switched our rendering process to pdflatex, which has proven more reliable for these cases.

Dataset Updates:
UniMER-1M/images: Replaced 2,451 files.
UniMER-Test/spe: Updated 26 files.
UniMER-Test/cpe: Corrected 81 files.

The updated dataset is now available for download directly from OpenDataLab or Hugging Face.
